etna/packages/magby/tests/test_Magby.py

```diff
@@ -0,0 +1,95 @@
+import unittest
+from unittest import TestCase
+import configparser
+from requests import Session
+from magby.Magby import *
+from testUtils import *
+
+
+url = 'https://magma.development.local'
+token = 'mytoken'
+currDir = os.path.dirname(os.path.realpath(__file__))
+
+
+class TestMagby(TestCase):
+    def setUp(self) -> None:
+        self.session = Session()
+        self.session.verify = False
+        self.vcr = prepCassette(self.session, os.path.join(currDir,'fixtures/cassettes'))
+        self.magby = Magby(url, 'mytoken')
+
+    def test_url(self):
+        self.assertEqual(self.magby.url, url)
+        self.magby.url = 'changed_url'
+        self.assertNotEqual(self.magby.url, url)
+        self.assertEqual(self.magby.url, 'changed_url')
+
+    def test_token(self):
+        self.assertEqual(self.magby.token, token)
+        self.magby.token = 'changed_token'
+        self.assertNotEqual(self.magby.token, token)
+        self.assertEqual(self.magby.token, 'changed_token')
+
+    def test__constructPayload(self):
+        payload = self.magby._constructPayload(projectName='ipi',
+                                               modelName='sample',
+                                               recordNames=["IPIADR001.T1"],
```

Comment:
I'm not sure if the seeded development ipi data is real or fake, but for magmaR, we were weary of fixtures creating a route for exposing patient data to the public.  And thus, the example project was born.

I haven't looked too closely at what the ipi seeding code does, but perhaps we could seed with the example project instead and then switch your tests over to that project?

Reply:
Fair, but fixtures expose some random ID's those are not PHI, and no additional data to trace to the individual. I can change, but would prefer to populate my dev environment rather than testing agains prod

Reply:
hmm, the matrix fixture is even all `null`s so agree there's maybe not much data here at all... but still a potential route for future tests to expose things.

I'll bring up potential of seeding dev envs with the example project in standup tomorrow.

Reply:
Sure, but keep in mind - there are no "future tests" which we don't control. All the data is in fixtures, so users outside the project won't be able to extend testing beyond those fixtures

Reply:
Right. I was on your side of this before, Anton. Any thoughts @graft?

Reply:
I'll test agains the example project. Thanks

Reply:
updated fixtures agains Example project in the prod